spec/services/find_ancestors_service_spec.rb

```diff
@@ -0,0 +1,63 @@
+describe FindAncestorsService do
+  subject(:find_ancestors) { described_class.new(goods_nomenclature_item_id).call }
+
+  context 'when the search term is empty' do
+    let(:goods_nomenclature_item_id) { '' }
+
+    it 'returns no ancestors' do
+      find_ancestors
+
+      expect(find_ancestors).to eq([])
+    end
+  end
+
+  context 'when the search term is Fully qualified (10 chars)' do
+    let(:commodity) { create :commodity }
```

Comment:
How do you feel about not mixing dsl/non-dsl styles for method invocations? Happy if you both prefer this, though

```suggestion
    let(:commodity) { create(:commodity) }
```